mizar/daemon/interface_service.py

```diff
@@ -179,6 +228,16 @@ def _ConfigureTransitAgent(self, interface):
         self.rpc.update_agent_metadata(interface)
         self.rpc.update_ep(interface)
 
+    def _DeleteTransitAgent(self, interface):
+        self.rpc.load_transit_agent_xdp(interface)
```

Comment:
Why are you loading the transit agent after deleting the interface? This is guaranteed to fail. I think you meant an unload transit agent. Actually, once you delete the interface, unloading is probably not needed. Can you verify that the XDP program (transit Agent) will not remain after you delete the interface?